### PR TITLE
Azure setup instructions and k8s files

### LINKED ISSUE #75: cloud deployment/hosting for OpenDP App
- decide platform
- [x] azure credit information
---
- [x] docker images
  - [x] vue prod files
- [x] k8s basic spec
- [x] k8s / ingress / https / etc
- [x] existing code working w/o secrets
- [x] add nginx frontend
- [x] postgres image w/ persistent data 
- [x] env secrets

### PR DESCRIPTION
**Pull request by raprasad**
This PR contains basic instructions on how to generate a kubernetes (k8s) file for Azure deployment. In addition, several specific settings and K8s files were adjusted/created. Notably:
 - The k8s deployment serves the static files via nginx which required an nginx container as well as additional settings involving static files.

### CODE REVIEW

File: deploy/k8s_maker/templates/azure_k8s_01.yaml [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+  #
+  # WILL ALL GO INTO SECRETS FILE ....
+  POSTGRES_USER: "test_postgres_user"
+  DB_USER: "test_postgres_user"
+  #
+  POSTGRES_PASSWORD: "test_not_real"
+  DB_PASSWORD: "test_not_real"
+  #
+  SECRET_KEY: "SECRET_KEY!-ADD-REAL-KEY-HERE!--ADD-REAL-KEY!1234!"
+  CRYPTOGRAPHY_KEY: "CRYPTOGRAPHY_KEY!-ADD-REAL-KEY!1234!"
```
Comment (ecowan) [inline deploy/k8s_maker/templates/azure_k8s_01.yaml]: Are these values added manually before deploying?

File: deploy/k8s_maker/templates/azure_k8s_02_misc.yaml [hunk 2]
```diff
@@ -0,0 +1,65 @@
+#apiVersion: v1
```
Comment (ecowan) [inline deploy/k8s_maker/templates/azure_k8s_02_misc.yaml]: Should these lines be deleted?

File: deploy/k8s_maker/templates/azure_k8s_01.yaml (hunk 1 above)
Comment (raprasad) [reply deploy/k8s_maker/templates/azure_k8s_01.yaml]: Leaving this in as a reference of a slightly different k8s setup--Pod versus Deployment--for a reference. Moved it into a folder named "old"

File: deploy/k8s_maker/templates/azure_k8s_02_misc.yaml (hunk 2 above)
Comment (raprasad) [reply deploy/k8s_maker/templates/azure_k8s_02_misc.yaml]: deleted several templates, including this one

**Pull request merged**